### PR TITLE
Added technical articulation.

### PR DESCRIPTION
**Pull request by thanrobinson**

### CODE REVIEW

**Comment by thanrobinson** [comment]
[revise HII technical articulation](https://trello.com/c/sXW4igPH/154-revise-hii-technical-articulation)

File: README.md [hunk 1]
```diff
@@ -20,7 +23,16 @@ For any given task date between two available GPW input images, the estimated po
...
+    COASTAL_NAVIGATION_DISTANCE = 80000  # meters
+    OCEAN_BUFFER_DISTANCE = 300  # meters
+    GSW_OCCURRENCE_THRESHOLD = 40
+    WIDE_RIVER_MIN_WIDTH = 30  # meters
+    GSW_CONNECTED_PIXEL_MIN = 1024  # pixels
+    INDIRECT_DISTANCE = 15000  # meters
+    DECAY_CONSTANT = -0.0003
+    INDIRECT_INFLUENCE = 10
 
 ## Usage
```
Comment (saanobhaai) [inline README.md:49]: Need to add `overwrite` arg and any others present in base task (have a quick look at one of the other driver's README)

File: README.md [hunk 2]
```diff
@@ -2,14 +2,17 @@
 
 ## What does this task do?
 
-This task calculates the (unitless) "influence" of navigable waterways on the terrestrial surface as one of the key drivers for a combined [Human Influence Index](https://github.com/SpeciesConservationLandscapes/task_hii_weightedsum). "Influence" is a pressure score based on proximity to a navigable waterway. Coasts, wide rivers and lakes are considered navigable.
+This task calculates the (unitless) "influence" of navigable waterways on the terrestrial surface as one of the key drivers for a combined [Human Influence Index](https://github.com/SpeciesConservationLandscapes/task_hii_weightedsum). "Influence" is a pressure score based on proximity to a navigable waterway. Coasts, wide rivers and lakes are considered navigable if the meet key criteria related to the distance from a population center (for coastlines including that of the Caspian Sea) or based on width and connectivity (inland waters). These methods are adapted from the logic followed by [Venter et al. 2016](https://www.nature.com/articles/sdata201667)).
```
Comment (saanobhaai) [inline README.md]: "... if they meet..."

**Review by saanobhaai — CHANGES_REQUESTED** [review]
Also need to insert details from Adam when he sends them

**Pull request merged**